src/mavsdk/plugins/follow_me/follow_me.cpp

```diff
@@ -57,42 +57,44 @@ FollowMe::Result FollowMe::stop() const
 }
 
 std::ostream&
-operator<<(std::ostream& str, FollowMe::Config::FollowDirection const& follow_direction)
+operator<<(std::ostream& str, FollowMe::Config::FollowAltitudeMode const& follow_altitude_mode)
 {
-    switch (follow_direction) {
-        case FollowMe::Config::FollowDirection::None:
```

Comment:
@JonasVautherin Is there a way to keep these for now, but mark them as deprecated?
It would be fairly straight forward to map the directions to an angle as a float.

Reply:
There was a discussion about this and that was the first plan, but since MAVSDK v2 is being released, it was decided that breaking the compatability would be ok 🤔

https://github.com/mavlink/MAVSDK-Proto/pull/287#issuecomment-1132930474